content/en/docs/tutorials/security/apparmor.md

```diff
@@ -250,10 +250,11 @@ error: error executing remote command: command terminated with non-zero exit cod
 
 To wrap up, let's look at what happens if we try to specify a profile that hasn't been loaded:
 
+
+
+
 ```shell
 kubectl create -f /dev/stdin <<EOF
-```
-```yaml
 apiVersion: v1
 kind: Pod
 metadata:
```

Comment:
This isn't right. Use a [`code` shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) and change the `kubectl create` command to suit (look at other pages that use the shortcode to see how it works).